sklearnex/linear_model/linear.py

```diff
@@ -225,13 +232,14 @@ def _initialize_onedal_estimator(self):
     def _onedal_fit(self, X, y, sample_weight, queue=None):
         assert sample_weight is None
 
+        supports_multi_output = daal_check_version((2025, "P", 1))
         check_params = {
             "X": X,
             "y": y,
             "dtype": [np.float64, np.float32],
             "accept_sparse": ["csr", "csc", "coo"],
             "y_numeric": True,
-            "multi_output": True,
+            "multi_output": supports_multi_output,

```

Comment:
While this reads like a regression in capability, maybe I am missing part of the picture. Was multi-output previously supported, or only on underdetermined systems? @Vika-F

Reply:
It was technically supported, but if the Cholesky factorization or solve failed, it would either throw an error, or segfault, or corrupt memory, depending on how the PLU factorization goes.

Reply:
Gotcha, which is why there is the sklearn_after_onedal on RuntimeError. As of now this would block using it in the case that the Cholesky succeeds (which may also occur).  Since this is a bit of a gray zone, I would want the weigh-in of @napetrov or @syakov-intel .

Reply:
I think worth discussing at arch meeting - i don't have preference

Reply:
@david-cortes-intel I would say lets merge this PR after we add a ticket to the backlog to address it.

Reply:
@icfaust To clarify: the issue (potentially crashing multi-output regression) happens in older releases of oneDAL, but is solved in the current GH main branch, which should form part of the 2025.1 release.